.github/workflows/publish-docker-master.yml

```diff
@@ -15,6 +15,6 @@ jobs:
       # Build and test Docker image
       - run: |
           echo ${{ secrets.GHCR_TOKEN }} | docker login ghcr.io -u nanocurrency --password-stdin
-          docker pull ghcr.io/nanocurrency/mkdocs-material-insiders:latest
+          docker pull squidfunk/mkdocs-material:latest
```

Comment:
As it is an external repository, it is better to fix it to a specific commit id.

Reply:
Updated to squidfunk/mkdocs-material:9.4.10@sha256:01605a03397a654b74b9de3157f56915d1e075e2d3bd22fcf3fb82c443553c25